CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # go-ipfs changelog
 
-### 0.4.10 - 2017-06-18
+### 0.4.10 - 2017-06-27
+
+Ipfs 0.4.10 is a patch release that contains several exciting new features,
```

Comment:
do we use all lowercase or all caps, i.e. ipfs for the package and IPFS for the technology?